src/status_im2/contexts/onboarding/sign_in/view.cljs

```diff
@@ -1,11 +1,10 @@
 (ns status-im2.contexts.onboarding.sign-in.view
   (:require [utils.i18n :as i18n]
-            [status-im2.contexts.onboarding.common.background.view :as background]
             [status-im2.contexts.syncing.scan-sync-code.view :as scan-sync-code]))
 
 (defn view
   []
   [scan-sync-code/view
    {:title             (i18n/label :t/sign-in-by-syncing)
     :show-bottom-view? true
-    :background        [background/view true]}])
+    :background        [:<>]}])
```

Comment:
hmm, I think the other use of this component is also using a fragment as well. So I think we can just delete background property here, in the other use of this component and in the component `scan-sync-code/view`